### ISSUE TITLE
Clearing does not update count

### ISSUE THREAD

**Issue by CyberMew** [ProxymanApp/Proxyman#693]
### Proxyman version? (Ex. Proxyman 1.4.3)
2.12.0

### macOS Version? (Ex. mac 10.14)
all

### Steps to reproduce 
Click on Clear button

### Expected behavior
Requests on screen to be cleared, regardless of what sidebar (app, domain, pinned) we selected, and whether or not we also cleared the filtered the results.
Domain count to be updated correctly. If I am viewing Domains and I have 2 domain, I filter for 1 domain, I clear the screen, then domain count should be 1. If I did not filter, and I clear it, domain count will be 0. 

### Current behavior
Count does not updated at all. Clearing of filtered results seems fine, no hopefully no issue there.

### Screenshots (optional)
![image](https://user-images.githubusercontent.com/4715911/99162405-72a50000-2738-11eb-8ad3-f220a26b85f8.png)
![image](https://user-images.githubusercontent.com/4715911/99162480-3a51f180-2739-11eb-9cb6-8cc042de0f78.png)

**Comment by NghiaTranUIT** [comment]
It's correct behavior @CyberMew 😄 

The count number is the sub-nodes inside this folder. For instance, 616 is the number of the domain inside the Domains folder. 

It's not the number of the flow of the current view, which has the Clear button.

Therefore, if you hit the Clear button, it will clear all requests from the current List, not remove the domain folder.

--------------------
To clear 616, you may click on the Clear session button on the Top Navigation Bar 👍

**Comment by CyberMew** [comment]
Hi @NghiaTranUIT, the Domains shows 616, but the list is empty. I have some pinned domains but definitely not more than 10. Showing 616 in this case is considered correct behaviour? I am not expecting to see 616 (sub-nodes) since there is no way I can access them anymore.

**Comment by NghiaTranUIT** [comment]
Can you click on the Triangle arrow? It will expand the subnode 😄

**Comment by CyberMew** [comment]
Ah ok. Yes, it expanded, but it's blank. There are no entries inside. So is the app supposed to retain all the subnodes with nothing inside after clicking on Clear?

**Comment by NghiaTranUIT** [comment]
Yes, it's correct. The scope of the Clear button is just the selected node (on the left). It means it will clear all flows in the Middle Table. It doesn't affect other nodes (in the left panel).

If you click Clear Session, it will clear all, and the count number will reset to 0, since all nodes are wiped out.

<img width="510" alt="Screen_Shot_2020-11-18_at_10_19_05" src="https://user-images.githubusercontent.com/5878421/99478762-90b46f80-2987-11eb-9abc-f8245d983934.png">

I understand that it's a little confusing, but it's essential to separate it since many users do not want to clear all, they just need to clear the current table view.
